tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -143,9 +143,12 @@ bool VersionSupportInfo::SkipIRSensitiveTest() {
   }
   return false;
 }
-bool VersionSupportInfo::SkipDxil_1_1_Test() {
-  if (m_DxilMajor < 1 || m_DxilMinor < 1 || m_ValMajor < 1 || m_ValMinor < 1) {
-    WEX::Logging::Log::Comment(L"Test skipped because it requires Dxil 1.1 and Validator 1.1.");
+bool VersionSupportInfo::SkipDxil_Test(unsigned major, unsigned minor) {
```

Comment:
Moving the version from name to parameters makes the name a little ugly.  Could we use something more intuitive now, like SkipDxilVersion instead of SkipDxil_Test?